### PR TITLE
Dashboard buttons for Assessment and Essay are linked to their respective pages.

### PR DESCRIPTION
**Pull request by SafiaMalik**
Dashboard buttons for Assessment and Essay are linked to their respective pages except for Courses button, that i need to do it latter once get that page for courses.

### CODE REVIEW

File: teamA/lib/Views/dashboard.dart [hunk 1]
```diff
@@ -263,6 +266,7 @@ class TeacherDashboard extends StatelessWidget {
     );
   }
 
+  // Widget to build circular buttons
```
Comment (AlmostCarbon-14) [inline teamA/lib/Views/dashboard.dart:269]: Please remove this redundant comment

File: teamA/lib/Views/dashboard.dart [hunk 2]
```diff
@@ -301,7 +305,31 @@ class TeacherDashboard extends StatelessWidget {
         ),
         child: ElevatedButton(
           onPressed: () {
-            // Add button functionality here
+            // Button functionality based on title
+            if (title == 'Courses') {
+              /*  Navigator.push(
```
Comment (AlmostCarbon-14) [inline teamA/lib/Views/dashboard.dart:310]: Please remove this commented code if it's not necessary

Comment (SafiaMalik) [reply teamA/lib/Views/dashboard.dart:310]: i need this commented code for courses button to link with courses pages once i get it from Joneice

File: teamA/lib/Views/dashboard.dart (hunk 1 above)
Comment (SafiaMalik) [reply teamA/lib/Views/dashboard.dart:269]: done

File: teamA/lib/Views/dashboard.dart (hunk 2 above)
Comment (AlmostCarbon-14) [reply teamA/lib/Views/dashboard.dart:310]: Understood, please add some text saying as much so that way the others don't think it's a mistake

**Pull request closed without merging**